<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SFMWindow</class>
 <widget class="QDialog" name="SFMWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="openImglPushButton">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>280</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>openImgl</string>
   </property>
  </widget>
  <widget class="QLabel" name="saveImageNumberLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>283</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>saveImageNumber</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectFeatureComboBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadCamMatPushButton">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>250</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>loadCamMat</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
